script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_9_1.txt

```diff
@@ -137,62 +137,62 @@ The dogs head towards me.
 I can see countless lines on their bodies.
 <Page30>
 ―――But despite this,
-my paralyzed mind doesn't instruct me to fight nor run.
+my paralyzed mind doesn't instruct me to fight or to run.
 <Page31>
 The first of the two dogs leaps.
-Its speed is beyond that of a human.
+Its speed is incomparable to that of a human.
 The length of the 20-meter corridor was traversed in no more than two seconds.
 <Page32>
-The dog opens its mouth.
-Its fangs, which are aligned like saws, lunge towards my throat.
+The black dog opens its mouth.
+Its fangs, like rows of sawteeth, lunge for my throat.
 <Page33>
 Swift and precise.
 The moment I realize it's coming.
-The dog makes a gargling sound and its fangs pierce my throat.
+The dog's fangs sink into my throat with a wet squelch.
 I, Tohno Shiki, have died.
 <Page34>
 No. That can't be right.
-There's no way I would be killed from this, let alone die.
+There's no way I could be killed like this, no way I could die like this.
 People dying isn't enough to make me hesitate.
 ――――A hot summer day.
-A long time ago, or maybe it was seven years ago.
+A long time ago, or maybe it was seven years ago.//ずっと昔、それとも七年も前の話。
```

Comment:
This feels pretty weird. それとも does normally makes me think `or`, but this would read a lot better to me as something like `Already 7 years in the past`,  or `A story from a goneaway time, 7 long years ago`. Haven't changed it but curious on opinions.
> ずっと昔、それとも七年も前の話。

Reply:
I think in this case it's like defining this as being a more specific description.

A long time ago, or _more precisely_, seven years ago.
@yk1512 can correct me if I'm wrong.